scripts/load-balancer/add-study-to-lb.sh

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
+#
+# Setup the frontend, backend, and url mappings for a new study to the LB.
+
+set -e
+
+NAME="$(basename "$0")"
+if (( $# < 5 )); then
+  echo "usage: $NAME <GCP_PROJECT> <LB_NAME> <GAE_SERVICE> <STUDY_DOMAIN> <STUDY_IDENTIFIER>"
+  echo ""
+  echo "For STUDY_IDENTIFIER, choose a short concise name for the study."
+  echo "This identifier will be used to name various LB resources for the study."
+  exit 1
+fi
+
+PROJECT_ID="$1"
+REGION='us-central1'
+LB_NAME="$2"
+GAE_SERVICE="$3"
+STUDY_DOMAIN="$4"
+STUDY="$5"
+
+echo "using gcp project: $PROJECT_ID"
+echo "using region: $REGION"
+echo "using lb name: $LB_NAME"
+echo "using gae service: $GAE_SERVICE"
+echo "using study domain: $STUDY_DOMAIN"
+echo "using study identifier: $STUDY"
+echo ""
+
+echo_run() {
+  echo "+ $@"
+  "$@"
+  echo ""
```

Comment:
Would using `set -v` or `set -x` before line 33 and disabling the option (`set +x`) be an option here instead of `echo`, or does the output need to be on `stdout`?

Reply:
Doesn't have to echo to stdout. I can use `set -x` here.

Reply:
Hmm nevermind, `set -x` is not what I want here. I think I'll stick with `echo. Using `set -x` will echo every command, so it means it will also echo the `set +x` after executing the command, resulting in something like:
```
+ gcloud compute ...
+ set +x

+ gcloud compute ...
+ set +x
```

Reply:
Also, I believe the docs is saying both `-v` and `-x` prints to stdout. So I guess my approach with `echo` is pretty much the same in that regard.